snapcraft/_options.py

```diff
@@ -223,6 +224,16 @@ def __init__(self, use_geoip=False, parallel_builds=True,
         self._set_machine(target_deb_arch)
         self.__debug = debug
 
+    @property
+    def is_host_comatible_with_base(self):
+        codename = None
+        with suppress(errors.OsReleaseCodenameError):
+            codename = OsRelease().version_codename()
+            logger.debug('Running on {!r}'.format(codename))
+
+        # TODO support more bases
+        return codename == 'xenial' or codename == 'trusty'
```

Comment:
Ah, good call adding Trusty, forgot about that one.

Reply:
Yeah, I change the name of this `property` to reflect that.